workflow/local_docker_build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# This script can be used to locally check and debug
+# the linux build process outside of CI.
+# This should be run from the root directory `packaging/manual_docker_build.sh`
+
+set -e
+rm -rf CMakeCache.txt
+find . -name CMakeFiles -delete
```

Comment:
If the `CMakeFiles` directories that are found are not empty, which they typically won't be, this will give an error rather than deleting the directories. Using something like `find . -name CMakeFiles -print0 | xargs -0 rm -rf` would address this.